news/newsletters.py

```diff
@@ -4,16 +4,39 @@
 It's used to lookup the backend-specific newsletter name from a
 generic one passed by the user. This decouples the API from any
 specific email provider."""
+from django.db.models.signals import post_save
+from django.db.models.signals import post_delete
 from django.core.cache import cache
 
-from news.models import Newsletter, NewsletterGroup
+from news.models import Newsletter, NewsletterGroup, SMSMessage
 
 
-__all__ = ('clear_newsletter_cache', 'newsletter_field', 'newsletter_name',
-           'newsletter_fields')
+__all__ = ('clear_newsletter_cache', 'get_sms_messages', 'newsletter_field',
+           'newsletter_name', 'newsletter_fields')
 
 
 CACHE_KEY = "newsletters_cache_data"
+SMS_CACHE_KEY = "sms_messages_cache_data"
+SMS_MESSAGES = {
+    'SMS_Android': 'MTo3ODow',
```

Comment:
Just already had that and I'll likely remove it later. It's only really necessary when this first rolls out to avoid any dropped requests. I could do a data migration as well, but this just seemed easier based on what was already there.